src/v/kafka/server/tests/create_topics_test.cc

```diff
@@ -335,7 +326,7 @@ FIXTURE_TEST(read_replica, create_topic_fixture) {
         "namespace": "kafka",
         "topic": "test-topic",
         "partition_count": 32,
-        "replication_factor": 3,
+        "replication_factor": 1,
```

Comment:
why did you change the replication factor? Read replica topic can have a replication_factor that is different from the original topic's